### DISCUSSION TITLE
Is it possible to connect to the Milvus Cloud database REST API?

### DISCUSSION THREAD

**Discussion by andreibondarev** [milvus-io/milvus#23261] [Q&A and General discussion] [answered]
Is it possible to connect to the Milvus Cloud database REST API? I saw the instructions on how to connect to the RPC endpoint (port: 19538) but not the REST API (port: 9091).

I had whitelisted my computer's IP address, and tried running the following:

```
curl -X 'GET' "https://[MY-USER]:[MY-PASSWORD]@[MY-HOST].aws-us-west-2.vectordb.zillizcloud.com:9091/api/v1/health"
```

It hangs and never resolves. I've also tried using the HTTP client and passing username and password as the basic auth.

**Comment by shanghaikid** [comment, marked as answer]
yes, we will provide http api in the two weeks, stay tuned.
